zetaclient/zetacore/client_query_observer.go

```diff
@@ -167,6 +167,14 @@ func (c *Client) GetBTCTSSAddress(ctx context.Context, chainID int64) (string, e
 	return resp.Btc, nil
 }
 
+func (c *Client) GetTSS(ctx context.Context) (types.TSS, error) {
```

Comment:
I think we have a test suite for `client_query` and we can add unit test for this function